File: Exercicios_guanaraba/Com_Ruby/exercicios/ex008_Conversor_de_Medida.rb
#Excreva um programa que leia um valor em metros e o exiba convertido em centimetros e milímetros.

puts 'Inisira a metragem: '
metros = gets.chomp.to_f
centimetros = metros * 100
milimetros = metros * 1000


puts "#{metros}m são: #{centimetros}cm e #{milimetros}mm"

File: Exercicios_guanaraba/Com_Dart/bin/ex008_Conversor_de_Medida.dart
//Escreva um programa que leia um valor em metros e o exiba convertido
//em centimetros e milimetros.

import 'dart:io';

void main(){


print ('Insira a metragem');
double metros = double.parse(stdin.readLineSync());

double centrimetros = metros * 100;
double milimetros = metros * 1000;

print (metros.toString() + ' m são' + centrimetros.toString() + ' cm e ' + milimetros.toString() + ' mm');




}